index.js

```diff
@@ -68,7 +70,35 @@ function toggleColor(element) {
   }
 }
 
+element.addEventListener("click",function(event) {
+  toggleColor(element)
+})
+
+
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector(".like-button")
+
+likeButton.addEventListener("click", function(event){
+  const likes = document.querySelector('.likes')
+  const numberOfLikes = parseInt(likes.textContent)++
+  likes.textContent = `${numberOfLikes} Likes`
+})
 
 /***** Deliverable 3 *****/
+
+const newGoalForm = document.querySelector('#new-goal-form')
+
+newGoalForm.addEventListener("submit", function(event){
+  event.preventDefault()
+  const goalsArray = player.goals
+
+  const goal = {
+    id: player.goals[goalsArray.length -1].id++,
+    playerId: 1,
+    link: event.target.link.value,
+    description: event.target.description.value
+  }
+  renderGoal(goal)
```

Comment:
Well done! I like your solution for getting the goal ID - again, the `++` operator isn't quite right (just a plain ol' `+ 1` will do). 

Once we get to working with fetch, the server will take care of making the ID for us, so you won't have to worry about that on the frontend.